src/Components/ButtonCutUp/ButtonCutUp.js

```diff
@@ -1,9 +1,25 @@
 import React, { Component } from 'react';
 import styles from './buttonCutUp.module.css';
+import Button from '../Button/Button';
 
 function ButtonCutUp(props) {
-  d
-}
+  const text = 'Hello World, My Name Is Igor. Im from Russia! Do you know it?';
+  const WordsAmount = text.split(" ").length;
```

Comment:
variable naming should be `camelCase`

Reply:
Done
` [9b80014](/Betra/duchamp-it-up/pull/3/commits/9b80014e8053b62ef435246c75c557a71a505676)`

Reply:
Done 
9b80014e8053b62ef435246c75c557a71a505676